src/components/AdjustVaultForm.tsx

```diff
@@ -49,6 +58,67 @@ const AdjustVaultForm = () => {
 
   const isActive = vaultToAdjust?.vaultState === 'active';
 
+  const onMaxCollateralClicked = () => {
+    assert(collateralAction === CollateralAction.Deposit);
+
+    if (!collateralPurse) {
+      /* no-op */
+      return;
+    }
+
+    setCollateralDeltaValue(collateralPurse.currentAmount.value);
```

Comment:
I always read "delta" as a value that can be negative, like movement along a line. "change" sounds more like absolute value of the difference.

also it makes me nervous to be passing around the bigints instead of the typed amounts.